doc/btp_spec.txt

```diff
@@ -295,10 +295,14 @@ Commands and responses:
 					Scan_Rsp_len (1 octet)
 					Adv_Data (0-255 octets)
 					Scan_Rsp (0-255 octets)
+					Duration (4 octets)
```

Comment:
Is this related to limited advertising? If so it would be good to refactor Opcode 0x08 - Set Discoverable command/response parameters in this PR as well:
"0x00 = Off
0x01 = General Discoverable
0x02 = Limited Discoverable"

so that :
"0x00 = Off
0x01 = On"